README.md

```diff
@@ -21,6 +21,14 @@ Documentation is available at [https://rancher-sandbox.github.io/cos-toolkit-doc
+|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| green-x86_64  | [![Build cOS releases-green-x86_64](https://github.com/rancher-sandbox/cOS-toolkit/actions/workflows/build-releases-green-x86_64.yaml/badge.svg)](https://github.com/rancher-sandbox/cOS-toolkit/actions/workflows/build-releases-green-x86_64.yaml)    | [![Build cOS master-green-x86_64](https://github.com/rancher-sandbox/cOS-toolkit/actions/workflows/build-master-green-x86_64.yaml/badge.svg)](https://github.com/rancher-sandbox/cOS-toolkit/actions/workflows/build-master-green-x86_64.yaml)    | [![Build cOS nightly-green-x86_64](https://github.com/rancher-sandbox/cOS-toolkit/actions/workflows/build-nightly-green-x86_64.yaml/badge.svg)](https://github.com/rancher-sandbox/cOS-toolkit/actions/workflows/build-nightly-green-x86_64.yaml)    | [![Build cOS Examples-green-x86_64](https://github.com/rancher-sandbox/cOS-toolkit/actions/workflows/build-examples-green-x86_64.yaml/badge.svg)](https://github.com/rancher-sandbox/cOS-toolkit/actions/workflows/build-examples-green-x86_64.yaml) |
```

Comment:
Examples now runs on master, better have that displayed:

[![Build cOS Examples-green-x86_64](https://github.com/rancher-sandbox/cOS-toolkit/actions/workflows/build-examples-green-x86_64.yaml/badge.svg?branch=master)](https://github.com/rancher-sandbox/cOS-toolkit/actions/workflows/build-examples-green-x86_64.yaml)